### PR TITLE
Fix Cronos address data

### PR DESCRIPTION
**Pull request by hewigovens**
It was missing from `AnyAddress.h`, tests added

### CODE REVIEW

File: src/Cardano/AddressV3.h [hunk 1]
```diff
@@ -91,8 +91,3 @@ inline bool operator==(const AddressV3& lhs, const AddressV3& rhs) {
 }
 
 } // namespace TW::Cardano
-
-/// Wrapper for C interface.
-struct TWCardanoAddress {
```
Comment (rsrbk) [inline src/Cardano/AddressV3.h:96]: Is this change intended to be here?

**Review by rsrbk — APPROVED** [review]

File: src/Cardano/AddressV3.h (hunk 1 above)
Comment (hewigovens) [reply src/Cardano/AddressV3.h:96]: This interface is not used at all, I forget to delete it in previous pr

**Review by catenocrypt — APPROVED** [review]
